src/NuGet.Commands/RestoreGraph.cs

```diff
@@ -101,13 +105,16 @@ public static RestoreTargetGraph Create(bool writeToLockFile, IEnumerable<GraphN
 
             var conflicts = new Dictionary<string, HashSet<ResolverRequest>>();
             var downgrades = new List<Tuple<GraphNode<RemoteResolveResult>, GraphNode<RemoteResolveResult>>>();
+            var versionConflicts = new List<Tuple<GraphNode<RemoteResolveResult>, GraphNode<RemoteResolveResult>>>();
```

Comment:
Code cleanup suggestion: Why not just have these be `VersionConflictResult`, `DowngradeResult` etc? The `Tuple` doesn't do anything useful.